src/test/scala/tech/sourced/siva/test/SivaReaderSpec.scala

```diff
@@ -13,8 +13,6 @@ import tech.sourced.siva.SivaReader
 import scala.collection.JavaConverters._
 
 class SivaReaderSpec extends FlatSpec with Matchers {
-  private val filenames = "gopher.txt" :: "readme.txt" :: "todo.txt" :: Nil
-  private val filenamesDeleted = "readme.txt" :: "todo.txt" :: Nil
   private val filenamesFolders =
```

Comment:
If you moved all the private values to the companion object move filenamesFolders too.